lib/galaxy/jobs/__init__.py

```diff
@@ -1743,7 +1743,14 @@ def _change_ownership( self, username, gid ):
         job = self.get_job()
         # FIXME: hardcoded path
         external_chown_script = self.get_destination_configuration("external_chown_script", None)
-        cmd = [ '/usr/bin/sudo', '-E', external_chown_script, self.working_directory, username, str( gid ) ]
+
+        cmd = [ '/usr/bin/sudo', '-E', ]
+        for v in ["PATH", "LD_LIBRARY_PATH", "PKG_CONFIG_PATH"]:
```

Comment:
Hmm, could you ask your admins to make an exception for the external_chown_script.py ?
If I look at https://unix.stackexchange.com/a/13246 it appear you can specify the env vars to keep for a specific path. In a sense you're already circumventing this policy by injecting the the environment variables anyway ... . If not I guess we can again make the list of env vars configurable in the galaxy.ini file, I think.